paddle/operators/detail/send_recv_impl.h

```diff
@@ -89,6 +90,58 @@ class RPCClient {
   std::unique_ptr<SendRecvService::Stub> stub_;
 };
 
+inline void SerializeToMessage(const std::string &name,
```

Comment:
Should not be inline, this function is too long.

Reply:
I think inline is a simple way, otherwise, we need a library for SerializeToMessage/DeserializeFromMessage.